plugins/filetypes/json/read.go

```diff
@@ -0,0 +1,38 @@
+package json
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+const maxJsonSize = 1024 * 1024 * 20
```

Comment:
I'm assuming this is the biggest that a single JSON row is allowed to be? We should probably document this as a comment, or perhaps later make it configurable. 

We will also need to deal with the case when this size is exceeded, because it will happen pretty soon. We had the same problem when reading JSON logs. There we stopped using a `bufio.Scanner` and instead used `bufio.Reader`. Maybe we can find "inspiration" from that implementation: https://github.com/cloudquery/plugin-sdk/blob/main/clients/log_reader.go#L33

The `Read` function is only used in testing right now though, so not a blocker by any means.